code/obj/item/pens_writing_etc.dm

```diff
@@ -1122,11 +1122,11 @@
 /* =============== STICKY NOTES =============== */
 
 /obj/item/postit_stack
-	name = "stack of crappy old sticky notes"
-	desc = "A little stack of notepaper that you can stick to things. These are the old ones that suck a lot."
+	name = "SHOULDN'T BE SEEING THIS"
+	desc = "OLD AND BAD"
```

Comment:
then delete it?

Reply:
check out my comments above, I requested this unless you wanna run through the rest of the maps indev both on and off the repo before merging